docs/contributors/advanced/review-a-merge-proposal.md

```diff
@@ -1,240 +1,171 @@
 
-  * Bad things happen -- check upstream to see if a release has been withdrawn
-    or needs an immediate fix-up due to unintentional breakage.
+* When merging from Debian or upstream:
+  * Check if there are newer versions to incorporate.
+  * Check if a release has been withdrawn or needs an immediate fix due to unintentional breakage.
 
-* Ensure that any changes in Debian do not imply that we need to update the
-  delta we carry (do not be fooled if it applies cleanly).
+* Ensure that any changes in Debian do not imply the need to update the {term}`Ubuntu delta` (even if the delta applies cleanly).
 
-* Ensure update maintainer has been run.
+* Ensure the {command}`update-maintainer` tool has been run.
 
 
-### Check the old delta
+### Old delta
 
-* Ensure that everything we dropped really can be dropped.
+* Ensure that everything dropped from the {term}`Ubuntu delta` really can be dropped.
 
-  * Vice versa, there may be even more that could be dropped (often needs a
-    little test to verify, which is suggested to the owner of the MP and only
-    rarely done by the reviewer).
+* Ensure that everything that _can_ be dropped really is dropped (this often needs a test to verify, which is suggested to the owner of the MP and only rarely done by the reviewer).
 
 
-### Check the new delta
+### New delta
 
-* Do the patches [follow DEP-3](http://dep.debian.net/deps/dep3/)?
+* Do the patches follow the {ref}`dep-3-patch-file-headers` format? (Use {command}`dep3changelog` to verify the headers and generate a changelog entry.
 
-  * You can use `dep3changelog`, both to verify the headers and to generate a
-    changelog entry.
-
-* Does the patch content and name {ref}`follow our additional style choices <work-with-debian-patches>`?
-
-  * This is not a strict check, but consistency helps to maintain packages together.
+* Does the patch content and name {ref}`follow our additional style choices <work-with-debian-patches>`? (This is not a strict check, but consistency helps to maintain packages together.)
 
 * Do the patches match what is (proposed) upstream?
 
-  * To be clear, if a certain case asks for violating this rule it is OK, but it
-    should be a conscious decision and not an accident.
-
-  * The intention is to avoid deviating the code we maintain too much from how
-    the project continues to evolve (hard to maintain well over long periods).
+  * If a certain case asks for violating this rule, it's OK, but it should be a conscious decision and not an accident.
 
-* Are the patches applied the right way according to `debian/source/format`?
+  * The intention is to avoid deviating the code we maintain too much from how the project continues to evolve (hard to maintain well over long periods).
 
-* Are all changes either:
+* Are the patches applied the right way according to `debian/source/format`? (See also {ref}`patches`.)
```

Comment:
```suggestion
* Are the patches applied the right way according to `debian/source/format`? (See also {ref}`patches`).
```

Reply:
I believe this was correct. The parentheses begin after the end of the preceding sentence (after the question mark), and they contain a whole sentence => the period should be inside.